### DISCUSSION TITLE
Increase DMG taken from skipping part of map MA_EP7 from 500 to 3000 HP

### DISCUSSION THREAD

**Discussion by leeroytm** [TitanTF/TitanTF#31] [General]
Suggestion by : masterfusion#7612 <@267283810746368001>
Located : https://discord.com/channels/774245035615977473/845235964712583208/849361622019211314

---
Description: Right now there are people who just jump there with max jump height and skip doing the part after opening the building up until the challenging part

**Comment by Lecubon1** [comment]
if i remembered correctly, there is a zone plugins (indicated by the white border) that prevented skipping before it was removed entirely
